src/main/java/org/radarcns/management/web/rest/OAuthClientsResource.java

```diff
@@ -80,20 +93,110 @@
     @Autowired
     private HttpServletRequest servletRequest;
 
+    private static final String ENTITY_NAME = "oauthClient";
+    private static final String PROTECTED_KEY = "protected";
+
+    /**
+     * GET /api/oauthclients
+     *
+     * Retrieve a list of currently registered OAuth clients.
+     *
+     * @return the list of registered clients as a list of {@link ClientDetailsDTO}
+     */
     @GetMapping("/oauthclients")
```

Comment:
I guess `oauth-clients` is the correct convention.

Reply:
or even just `clients`?

Reply:
Hmm, i think `clients` is too general, with `oauth-clients` there is no ambiguity as to what we are referring to.